app/pages/delete-order/manifest.json

```diff
@@ -3,6 +3,10 @@
   "title": "Delete order {{orderId}}?",
   "description": "Confirm you want to delete this order.",
   "orderDescriptionTitle": "Order description",
-  "noButtonText": "No",
-  "yesButtonText": "Yes"
+  "noButton": {
```

Comment:
Does this need to be an object?

Reply:
no it doesn't but I started with resizing and was about to put the width in there. But then the content got changed so didn't need the width. but I actually prefer it as an object. gives us bit more scope to add more config to the button.